test/e2e/app-dir/use-cache/app/(no-suspense)/[id]/page.tsx

```diff
@@ -4,7 +4,10 @@ async function getCachedRandom(n: number) {
 }
 
 export async function generateStaticParams() {
-  return [{ id: await getCachedRandom(10) }, { id: await getCachedRandom(2) }]
+  return [
+    { id: `a${await getCachedRandom(9)}` },
+    { id: `b${await getCachedRandom(2)}` },
```

Comment:
Added a prefix character here to avoid collapsing into a single page when both random numbers are equal, which would make the added test flaky.